src/testUtils/mocks/dataSources.ts

```diff
@@ -1,4 +1,5 @@
 /* istanbul ignore file */
+
```

Comment:
for some reason this file started missing coverage without this space here locally